Item.cpp

```diff
@@ -0,0 +1,49 @@
+// Item.cpp
+// Sheng Fang, Rabiya Majeed, & Zulaikha Zakiullah
+// This instantiates the item class. Item stats should be read off a txt. file (?)
+
+#include "Item.h"
+
+Item::Item(std::string st) {
+
+	// Splits string by words and assigns the stats from array
+	std::stringstream ssline(st);
+	std::string* stats = new std::string[5]{};
+
+	int i = 0;
+
+	while (ssline.good() && i < 5) {
+		ssline >> stats[i];
+		++i;
+	}
+
+	type = stats[0];								// Specifies what class of item it is

```

Comment:
Does it work without doing Item::type? Same for the others below

Reply:
Do you mean without Item::type = stats[0];? I think so, it doesn't show any errors.

Reply:
Oh lol, didn't know that. I'll have to go back and change the other classes then since this is shorter